pymc3/distributions/continuous.py

```diff
@@ -910,10 +913,10 @@ def logcdf(self, value):
         """
         sigma = self.sigma
         z = zvalue(value, mu=0, sigma=sigma)
-        return tt.switch(
```

Comment:
The logic here was to have numerically stable `erfc` function.
Indeed not relevant for reasonable inputs for the `logcdf` of a `HalfNormal`, but maybe useful to abstract away into another function?

Reply:
Thanks for clarifying the purpose. Do you know of any places in the existing codebase that could benefit from that more stable `erfc`?  If you think it is useful for users directly as well, would you like to implement a PR for it?

Reply:
NVM: I think it already exists in `normal_lcdf` https://github.com/pymc-devs/pymc3/blob/b7daf266ef4247e33b32db069800180627bd731c/pymc3/distributions/dist_math.py#L128-L135